Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Company/Company.tsx

```diff
@@ -81,6 +80,14 @@ export default function Company (props: { CompanyID: number }) {
             cache: true,
             async: true
         });
+
+        handle.done(() => {
+            window.location.href = homePath + 'index.cshtml?name=Companies'
```

Comment:
Can you confirm this will actually take you to the correct page once the Company is deleted (I think previously it would not)

Reply:
Yes, I just tried it and it takes you to the correct page, there was previously a typo in the path